Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -4259,7 +4259,7 @@ namespace AssetProcessor
                         AssetBuilderSDK::SourceFileDependency::SourceFileDependencyType::Wildcards
                         ? SourceFileDependencyEntry::DEP_SourceLikeMatch
                         : JobDependencyType,
-                        !entry.m_sourceFileInfo.m_uuid.IsNull(),
+                        !jobDependency.m_jobDependency.m_sourceFile.m_sourceFileDependencyUUID.IsNull(),
```

Comment:
Q: Why do we set as bool when the underlying type is bit flags as AZ::u32?

https://github.com/o3de/o3de/blob/f4ed5e8efd39a4b93e8f7e1eed445dcf33f1923c/Code/Framework/AzToolsFramework/AzToolsFramework/AssetDatabase/AssetDatabaseConnection.h#L218

Reply:
I'm not actually sure, I know the database doesn't support bools, its possible while this API was being developed it was originally a bool and then got switched since the DB doesn't support bool.  I think we could probably change this constructor to accept a bool without an issue though.